src/core/editform/qgsattributeeditorelement.h

```diff
@@ -143,12 +144,69 @@ class CORE_EXPORT QgsAttributeEditorElement SIP_ABSTRACT
      */
     void setShowLabel( bool showLabel );
 
+    /**
+     * Returns the custom label font, this is only effective if overrideStyleLabel() is also set.
+     * \see setLabelFont()
+     * \see overrideLabelStyle()
+     * \see setOverrideLabelStyle()
+     * \since QGIS 3.26
+     */
+    const QFont &labelFont() const;
```

Comment:
QFont/QColor are implicitly shared, so we should return values instead of references here